src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/ConfigurationDebuggerView.xaml

```diff
@@ -152,17 +174,334 @@
+																	  ContentTranslationOffset="40"
+																	  OffsetFromCenter="{Binding TemplateSettings.DropDownOffset, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+																	  OpenedLength="{Binding TemplateSettings.DropDownOpenedHeight, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+																	  OpenedTargetName="PopupBorder" />
+										</Storyboard>
+									</VisualState>
+								</VisualStateGroup>
+							</VisualStateManager.VisualStateGroups>
+
+							<Grid x:Name="ComboBoxContent"
+								  Padding="{TemplateBinding Padding}">
+								<!--<Grid.Resources>
+									--><!--  Resources added here in order to manage the ContentPresenter TranslateY depending if there is a PlaceholderText or not  --><!--
+									<CompositeTransform x:Key="ContentPresenter_CompositeTransformWithPlaceholder"
+														TranslateY="{Binding SelectedItem, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource NullToContentTranslateYConverter}, TargetNullValue=0, FallbackValue=0}" />
+
+									<CompositeTransform x:Key="ContentPresenter_CompositeTransformWithoutPlaceholder"
+														TranslateY="0" />
+
+									<um:FromEmptyStringToValueConverter x:Key="EmptyToCompositeTransformConverter"
+																		NotNullOrEmptyValue="{StaticResource ContentPresenter_CompositeTransformWithPlaceholder}"
+																		NullOrEmptyValue="{StaticResource ContentPresenter_CompositeTransformWithoutPlaceholder}" />
+								</Grid.Resources>-->
+
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition Width="Auto" />
+									<ColumnDefinition Width="*" />
+									<ColumnDefinition Width="Auto" />
+								</Grid.ColumnDefinitions>
+
+								<!--  ContentPresenter  -->
+								<ContentPresenter x:Name="ContentPresenter"
+												  Grid.Column="1"
+												  MaxLines="1"
+												  VerticalAlignment="Center"/>
+
+								<!--  Down  -->
+								<Path x:Name="DropDownGlyph_Down"
+									  Grid.Column="2"
+									  Width="10"
+									  Height="5"
+									  Margin="7,0"
+									  HorizontalAlignment="Center"
+									  VerticalAlignment="Center"
+									  Data="{StaticResource DownArrowPathData}"
+									  Fill="White"
+									  Stretch="Uniform" />
+
+								<!--  Up  -->
+								<Path x:Name="DropDownGlyph_Up"
+									  Grid.Column="2"
+									  Width="10"
+									  Height="5"
+									  Margin="7,0"
+									  HorizontalAlignment="Center"
+									  VerticalAlignment="Center"
+									  Data="{StaticResource UpArrowPathData}"
+									  Fill="White"
+									  Stretch="Uniform"
+									  Visibility="Collapsed" />
+							</Grid>
+
+							<!--  Popup  -->
+							<Popup x:Name="Popup">
+								<Border x:Name="PopupBorder"
+										Background="#DD000000">
+
+									<!--  Workaround for Issue #213 where android PopupBorder doesn't get redrawn properly, adds Background, CornerRadius and Border to ScrollViewer  -->
```

Comment:
What is this issue? Do we have a link? I feel like it would be easier track it.

Reply:
I don't know. It's a style I copied from Uno. I couldn't find the issue so I removed that comment.